libraries/iterate/src/index.js

```diff
@@ -133,19 +139,15 @@ class Iterate extends events.EventEmitter {
       const duration = d[0] * 1000 + d[1] / 1000000;
 
       this.emit(iterError ? 'iteration-failure' : 'iteration-success');
-      if (this.monitor) {
-        this.monitor.info('iteration', {
-          status: iterError ? 'failed' : 'success',
-          duration,
-        });
-      }
+
+      this.monitor.log.periodic({
+        name: this.name,
+        duration,
+        status: iterError ? 'exception': 'success',
+        error: iterError ? serializeError(iterError) : undefined,
+      }, {level: iterError ? 'err' : 'notice'});
 
       if (iterError) {
-        if (this.monitor) {
-          this.monitor.reportError(iterError, 'warning', {
-            consecutiveErrors: failures.length,
-          });
-        }
```

Comment:
What's the rationale for not reporting these errors as errors anymore?  Won't that mean they don't appear in error dashboards like sentry?

Reply:
Ah, I had something in my head about wanting every "event" in tc to end up with a single log message but we could duplicate it into an error event too. I think actually this would work with both because it sets the log level to error. Does that make sense or should we duplicate?

Reply:
I think we should have two events.  Stackdriver would (maybe, who knows) find the error in the output, but maybe something else wouldn't.  For example, I was thinking of connecting to Sentry directly from the `monitor.reportError` method, rather than pulling errors out of the log stream and calling the Sentry API with them.

One option here is to not include the serialized error in the iterate/oneShot output, since it's just intended for monitoring -- that reduces the conceptual duplication a bit.  And it means that an ops person who starts seeing failures has a time-range to go start searching for errors.

Reply:
Could we hook in direct sentry integration at the logging level rather than monitor and have anything with > err be sent to sentry or something? I'm not firmly in either camp yet. Just trying to figure what's best.

Is there a way you'd prefer this to be, @sciurus / @edunham

Reply:
Possibly -- I just don't want to tie us too closely into the stackdriver model.

Reply:
I don't think as written this would end up in stackdriver error reporting, since it doesn't conform to the format at https://cloud.google.com/error-reporting/docs/formatting-error-messages

I do think you want to make sure the serialized error makes it to sentry or stackdriver error reporting, so that you can have grouping.

I think Emily and I are agnostic on how this is implemented internally, and the only thing we know we don't want is to have to run a separate process to tail the logs, extract errors, and submit them somewhere.